### PR TITLE
Add Complexity handling practices as one tip

### PR DESCRIPTION
**Pull request by chunfuwen**
This tip is borrowed from Complexity part in Google engineering practice[1].

[1]https://google.github.io/eng-practices/review/reviewer/looking-for.html

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
@smitterl 
@dzhengfy 
Please help take a look. 
This practice is follow-up action on our PR review meeting

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by smitterl — CHANGES_REQUESTED** [review]
@chunfuwen Thank you for this addition! I think there should be some change in format and optionally in style. Please, check my comment. Thank you!

File: tp-libvirt_review_comment_summary.rst [hunk 1]
```diff
@@ -120,6 +120,9 @@ Coding
 
  - instead of hardcoding libvirt version switches in the test scripts, it's preferable to add the version that's minimally supported in the test configuration file, e.g. called `func_supported_since_libvirt_ver = (7, 0, 0)` in the cfg and use in the script as `libvirt_version.is_libvirt_feature_supported(params)`.
 
+- Complexity handling
+  - Encourage contributors to solve the problem they know needs to be solved now, not the problem that the contributor speculates might need to be solved in the future. The future problem should be solved once it arrives and you can see its actual shape and requirements in the physical universe.
```
[the remark this answers is not in the document]
Comment (chunfuwen) [reply tp-libvirt_review_comment_summary.rst]: @smitterl , updated, please double check this

**Comment by chunfuwen** [comment]
@smitterl 
Updated almost like https://github.com/chunfuwen/tp-libvirt/pull/1, but remain qualifying words, which is already there in original statement ]https://google.github.io/eng-practices/review/reviewer/looking-for.html.

**Review by smitterl — APPROVED** [review]

**Comment by chunfuwen** [comment]
Let's merge it since all reviewers approve this

**Pull request merged**